x/ecocredit/spec/01_concepts.md

```diff
@@ -11,6 +11,17 @@ A credit class is the primary abstraction for ecosystem service credits and is d
 
 Once a new credit class is created, credits can be issued at-will in distinct batches by any address in the **issuer list** of the corresponding credit class.
 
+## Project
+
+A project describes the high-level on-chain information for an issuer associated with a credit class. A project is defined by:
+- **project ID**: The unique ID of the project
+- **name***: The name of the project
+- **class ID**: The unique ID of the credit class for this project
+- **project location**: The location of the project backing the credits issued from this project
+- **metadata**: A byte array (up to 256 bytes) which can be used to store small amounts of metadata, or a URI that points to an off-chain resource for querying more complete metadata information. This usually would include descriptive information about the project.
+
+Each credit batch is associated with a project, backing each issuance with information about the issuer of the credits. A credit class can have multiple projects that issue credits from the credit class.
```

Comment:
This framing is not really what I understand to be the model we are aiming for. In my understanding, projects are not describing issuers. They are separate entities (issuers being closer to verifiers, and projects being eventually administered by project developers/land stewards).  So I think it'd be more accurate to say something like "backing each issuance with information about the on-the-ground project to which the credits were issued."


I just scheduled some time on Monday to go over this with @S4mmyb and @ryanchristo so hopefully we can all get together then and make sure we're understanding things the same way.